config/rbac/katamonitor.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: prometheus-k8s
+  namespace: openshift-sandboxed-containers-operator
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - services
+  - endpoints
+  - pods
+  verbs:
+  - get
+  - list
+  - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: prometheus-k8s
```

Comment:
Usually we add a `binding` suffix to to distinguish with the role name. For example `prometheus-k8s-binding`.
Having said that, how about having the role name as `monitor-role` and rolebinding name as `monitor-rolebinding` ?
Will be in sync with the other roles and rolebindings .

Reply:
Sure, I will change that.